app.py

```diff
@@ -136,12 +139,14 @@ def list_users_():
     @app.route("/manager/create_user", methods=["POST"])
     @authenticated(ADMINS_GROUP)
     @csrf_needed
+    @valid(json=CreateUser)
```

Comment:
nitpick: I would use `validated` to be consistent with other decorators

Reply:
Nice catch, thanks! Coming up in next revision!